setup.py

```diff
@@ -11,14 +11,16 @@
 from setuptools.command.test import test as TestCommand
 from setuptools.command.build_ext import build_ext as BuildExtCommand
 
-if sys.version_info < (3, 6):
+min_version = (3, 6)
+
```

Comment:
Did you want to just define `min_ver_str = '.'.join(str(n) for n in min_version)` here, and use that below?  You could then use f-strings:
`f"...Python {min_ver_str} or above is required"

Reply:
Would be slightly better. I‘ll wait a day or so to see if there‘s feedback. Otherwise I will merge as is. You can always do a follow up if you feel inclined.

Reply:
Or
```
min_version = distutils.version.LooseVersion("3.6.0")
if platform.python_version() < min_version: ...

...

python_requires=">={}".format(min_version)
```